lib/pbench/cli/agent/commands/tools/clear.py

```diff
@@ -129,17 +139,19 @@ def callback(ctxt, param, value):
         "-g",
         "--group",
         default="default",
-        required=True,
         expose_value=False,
         callback=callback,
-        help="list the tools used in this <group-name>",
+        help=(
+            "Clear the tools in the <group-name> group.\n"
+            "If no group or name is specified, the 'default' group is assumed."
+        ),
     )(f)
 
 
 def _name_option(f):
     """Tool name to use"""
 
-    def callback(ctxt, param, value):
+    def callback(ctxt, _param, value):
         clictxt = ctxt.ensure_object(CliContext)
         clictxt.name = []
```

Comment:
Why do we use the list here?, we could just do (like we do with group option)
```
clictxt = ctxt.ensure_object(CliContext)
clictxt.name = value
return value
```
and then on line 52 we can do `names = self.context.name.split(",")` that way we support the comma separated `name` option

Reply:
OK, you convinced me...I replaced the list with a direct assignment.  😀